aspnetcore/mvc/advanced/custom-model-binding/samples/2.x/CustomModelBindingSample/README.md

```diff
@@ -1,12 +0,0 @@
-# Custom Model Binding Demo
```

Comment:
A couple of comments in this were out-of-date/incorrect. I was going to fix them up but I concluded that this isn't needed for an advanced sample. I've also not seen any other README's that list out endpoints and instructions.

Reply:
The README files aren't localized, so Tom suggested moving content into topics. I've been doing so as I come across them on sample updates.

Reply:
Thanks @guardrex. That's helpful. I don't think there's anything else that needs to be brought across in this example, but let me know if you disagree.

Reply:
No worries ... I was just mentioning that in passing. Perhaps, the new [samples site](https://docs.microsoft.com/en-us/samples/browse/?products=aspnet-core) has something to make the experience better so that a cross-link to a sample there will load localized content automagically ✨.